### ISSUE TITLE
Real time stereo camera disparity map

### ISSUE THREAD

**Issue by Kheiden** [Kheiden/RPi-tankbot#45]
Depends on #44

I want to create a real time stereo camera disparity map.

**Comment by Kheiden** [comment]
I'm looking for a minimum of 2 FPS for the realtime disparity map stream, but the stretch goal is 15 FPS.

**Comment by Kheiden** [comment]
The fastest that I can get the stream to go is 4.46 seconds per frame, so 7 frames in a 30 second time period.  If I want to go any faster than this, I will need to incorporate a multi-threaded approach.

**Comment by Kheiden** [comment]
I need to implement a warmup override feature to the take_stereo_photo method.

**Comment by Kheiden** [comment]
I want the robot to take 8 3d cloud points while rotating 45 degrees between 3d cloud points.  After they are taken, I want the robot to generate a MeshLab Project file which specifies the amount of rotation between each of the 3d cloud point files.

**Comment by Kheiden** [comment]
Even after overriding the camera warmup, I'm only able to take 9 disparity map photos in 30 seconds. I'm going to test the camera to quantify how many frames I can capture in 30 seconds.

**Comment by Kheiden** [comment]
I'm not able to process the disparity maps fast enough. I'll have to utilize client/server architecture to send the L and R photos from the RPi to my laptop, calculate the disparity map, then send the disparity map back to the RPi.
